schema/bom-1.4-strict-SNAPSHOT.schema.json

```diff
@@ -1074,6 +1082,278 @@
           "description": "The value of the property."
         }
       }
+    },
+    "advisory": {
+      "type": "object",
+      "title": "Advisory",
+      "description": "Location of additional advisory information.",
+      "required": ["url"],
+      "additionalProperties": false,
+      "properties": {
+        "title": {
+          "type": "string"
+        },
+        "url": {
+          "type": "string",
+          "format": "iri-reference"
+        }
+      }
+    },
+    "cwe": {
+      "type": "integer",
+      "minimum": 1,
+      "title": "CWE",
+      "description": "Integer representation of a Common Weaknesses Enumerations (CWE). For example 399 (of https://cwe.mitre.org/data/definitions/399.html)"
+    },
+    "severity": {
+      "type": "string",
+      "title": "Severity",
+      "description": "Textual representation of the severity of the vulnerability adopted by the risk analysis method. If an other risk analysis method is used other than whats defined in scoreSourceType, the user is expected to translate appropriately to match with an element value below.",
+      "enum": [
+        "critical",
+        "high",
+        "medium",
+        "low",
+        "none",
```

Comment:
@luhring `none` is intended to be informational. Perhaps it should change to `info`? In some cases with previously unknown vulnerabilities against services, there's a need to report a informational finding that isn't technically a vulnerability. For example, the absence of Content Security Policy on a website to mitigate XSS, but no XSS was actually discovered. It's an informational warning.

I don't think it would ever be used for components however. I cannot see a use case for `info`, but perhaps there is one?

Reply:
I don't have much of an opinion on this one. I just wanted to note that in some contexts, there might be a negligible (pun intended!) mis-translation from scanners' upstream data sources to the final report in CycloneDX, in the case where the upstream source didn't actually say "none". I think this is an unimportant distinction, for the most part, so there might be no need to change this.

Reply:
I've added an 'info' category in addition to the ones supported.